src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -42,6 +42,8 @@ public int getStreamingFlagValue() {
 
     private final long mServiceConnectionTimeout;
 
+    private final KvsRetryStrategyCallbacks mKvsRetryStrategyCallbacks;
```

Comment:
The ClientInfo in the PIC layer has https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/65e38dac9b30523d43a57bc009d679e627b58d9a/src/client/include/com/amazonaws/kinesis/video/client/Include.h#L1197-L1201

```
   // Retry strategy for the client and all the streams under it
    KvsRetryStrategy kvsRetryStrategy;

    // Function pointers for application to provide a custom retry strategy
    KvsRetryStrategyCallbacks kvsRetryStrategyCallbacks;
```
Any reason why we are only introducing one of these instead of both?